Kommunicate-io/Kommunicate-Web-SDK
# Allow file upload with html in their name and Escape the HTML tags when showing in UI

### What do you want to achieve?
- Allow to upload files with html in their names. While rendering it in the UI, escape the HTML so it shown normally and not rendered as HTML.

### PR Checklist


### How was the code tested?

- Upload a file named `<h1> test <h1>.txt` and `<h1> test <h1>.png`.
- Check that they are uploaded normally
- HTML should not be rendered

<img width="1177" alt="Screenshot 2020-10-23 at 12 32 07 PM" src="https://user-images.githubusercontent.com/9497860/96966646-092c2b80-14c3-11eb-8711-ce88ffb54a55.png">
<img width="1177" alt="Screenshot 2020-10-23 at 12 32 12 PM" src="https://user-images.githubusercontent.com/9497860/96966656-0e897600-14c3-11eb-8031-be1b624fc2cb.png">

### What new thing you came across while writing this code? 
-

### In case you fixed a bug then please describe the root cause of it? 
-

NOTE: Make sure you're comparing your branch with the correct base branch
